pkg/motor/callback_test.go

```diff
@@ -0,0 +1,43 @@
+package motor
+
+import (
+	"testing"
+
+	"github.com/sonr-io/sonr/third_party/types/common"
+	mt "github.com/sonr-io/sonr/third_party/types/motor/api/v1"
+	"github.com/stretchr/testify/assert"
+)
+
+type testCallback struct {
+	WalletEventExec bool
+}
+
+func (tc *testCallback) OnDiscover(data []byte) {}
+
+func (tc *testCallback) OnWalletEvent(data []byte) {
+	tc.WalletEventExec = true
+}
+
+func (tc *testCallback) ClearState() {
+	tc.WalletEventExec = false
+}
+
+func (tc *testCallback) GetWalletEventState() bool {
+	return tc.WalletEventExec
+}
+
+func Test_Callbacks(t *testing.T) {
```

Comment:
Can this use the test suite?

Reply:
Will this be included in CI?

Reply:
Yeah +1 really want every test to be run-able in CI

Reply:
this is ci runnable and will run alongside the motor test suite. we can add it to the suite but it doesnt need the overhead contained in it.

Reply:
Sgtm